trackpy/feature.py

```diff
@@ -663,7 +676,7 @@ def batch(frames, diameter, minmass=100, maxsize=None, separation=None,
           percentile=64, topn=None, preprocess=True, max_iterations=10,
           filter_before=True, filter_after=True,
           characterize=True, engine='auto',
-          output=None, meta=True):
+          output=None, meta=None):
```

Comment:
Yes. Current default behavior: spew yaml files with automatically-generated file names. New default behavior: be quiet. Require the user to specify a file handle or filename.